frontend/app/src/services/types-api.ts

```diff
@@ -41,6 +47,20 @@ export interface SupportedAssets {
   readonly [key: string]: ApiSupportedAsset;
 }
 
+export interface Watcher {
+  readonly identifier: string;
+  readonly type: WatcherType;
+  readonly args: object;
```

Comment:
Are the args a different type than the interface below?

Reply:
Eventually different watcher types will have different args interfaces, so I was trying to not limit us to a specific structure for now.